### DISCUSSION TITLE
Prompt for custom password without requiring an account

### DISCUSSION THREAD

**Discussion by jpjonte** [goauthentik/authentik#3583] [Q&A] [answered]
Hi,

I'm curious if the following is possible with authentik:

I'd like to make a site accessible to anyone who has the password I give them, without them having to create an account in my authentik instance.
Basically like HTTP Basic Auth but with a pretty login page provided by authentik.

I've played around with flows, prompts, and policies, but so far it looks like it's not possible.

**Comment by BeryJu** [comment, marked as answer]
Not directly possible without an account but you can add a policy to a flow that sets a pending user, then add a password stage which checks against app passwords, and then use those passwords

**Comment by jpjonte** [comment]
That sounds like a good solution. Thanks!
